src/api.ts

```diff
@@ -309,7 +309,6 @@ async function getRemoteBranches(): Promise< Map< string, string > > {
 			} )
 		);
 
-		repo.free();
```

Comment:
Not a thing anymore:

> All free functions have been removed. The expectation is that they will be collected by the GC.

From https://github.com/nodegit/nodegit/blob/master/CHANGELOG.md#v0240-2019-01-16